Documentation/Getting-Started/storage-architecture.md

```diff
@@ -21,12 +21,83 @@ specified in the Ceph custom resources (CRs) and apply the changes.
 
 Rook automatically configures the Ceph-CSI driver to mount the storage to your pods.
 
-![Rook Components on Kubernetes](ceph-storage/kubernetes.png)
+![Rook Components on Kubernetes](ceph-storage/Rook High-Level Architecture.png)
```

Comment:
Lines 24-40 need to be removed. They are duplicated from the new diagram section.